### DISCUSSION TITLE
problemas na autenticação do QFieldCloud

### DISCUSSION THREAD

**Discussion by yamaciro** [opengisch/QField#2521] [Q&A]
Boa Tarde,

Instalei a versão 4.0.0-BETA18 do QFieldSync no Qgis 3.16.16-Hannover. Finalizei o cadastro no site http://app.qfield.cloud/ e criei login de acesso, mas não estou conseguindo autenticar o meu site QFieldCloud no Qgis.

At,

**Comment by suricactus** [comment]
Please write in English.
I see BETA18, please upgrade to BETA20 and let us know in English if it worked for you.
